src/OrchardCore.Build/Dependencies.props

```diff
@@ -26,6 +26,7 @@
     <PackageManagement Include="GraphQL.SystemTextJson" Version="4.8.0" />
     <PackageManagement Include="Jint" Version="3.0.0-beta-2049" />
     <PackageManagement Include="HtmlSanitizer" Version="8.0.692" />
+    <PackageManagement Include="libphonenumber-csharp" Version="8.13.18" />
```

Comment:
Please add new dependencies versions and licenses to src\docs\resources\libraries\Readme.md

Reply:
added.